src/plugin/context.rs

```diff
@@ -32,6 +32,8 @@ pub struct RapierContext {
     /// The island manager, which detects what object is sleeping
     /// (not moving much) to reduce computations.
     pub islands: IslandManager,
+    // FIXME: This shoulnd not be skipped.
```

Comment:
- Potential fix: https://github.com/dimforge/rapier/pull/675

Reply:
Looks like the fix is merged. Can the `serde(skip)` be removed now?